src/components/page-types/formPage/tripRelationshipCard.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useContext } from 'react';
+import { FlexBox } from '../../layout/FlexBox';
+import { GridCell } from '../../layout/GridCell';
+import { Heading } from '../../simple/Heading';
+import HeroIcon from '../../simple/heroIcon';
+import { FormContext } from '../../../contexts/FormContext';
+
+const TripRelationShipCard = ({ traveler }) => {
+  const [state, dispatch] = useContext(FormContext);
+  const [removeBtn, setRemoveBtn] = useState(false);
+
+  let fullName = '';
+  if (traveler && traveler?.digitalName) {
+    fullName = traveler.digitalName;
+  }
+  if (traveler && traveler?.firstName && traveler?.lastName) {
+    fullName = `${traveler?.firstName} ${traveler?.lastName}`;
+  }
+
+  const addRelationship = () => {
+    dispatch({
+      type: 'addTraveler',
+      payload: traveler,
+    });
+    setRemoveBtn(true);
+  };
+
+  const removeRelationship = () => {
+    dispatch({
+      type: 'removeTraveler',
+      payload: traveler?.id,
+    });
+    setRemoveBtn(false);
+  };
+
+  return (
+    <GridCell md={4}>
+      <FlexBox
+        direction="col"
+        className="icon-card su-bg-white print:su-hidden su-group su-relative su-block children:su-mx-auto su-text-center sm:su-max-w-[42rem] lg:su-max-w-[50rem] xl:su-max-w-full su-w-full su-mx-auto su-rs-px-3 md:su-rs-px-1 xl:su-rs-px-3 su-rs-py-3 xl:su-rs-py-4 su-basefont-23 su-break-words su-border su-border-solid su-shadow-sm hover:su-shadow-md"
+      >
+        <HeroIcon
+          iconType="play"
+          className="su-mb-02em su-transition-colors su-text-m2"
+        />
+        {removeBtn ? (
+          <span>
+            <HeroIcon
+              iconType="play"
+              className="su-transition-colors su-inline"
+            />
+            Remove
+          </span>
+        ) : (
+          <span>
+            <HeroIcon
+              iconType="play"
+              className="su-transition-colors su-inline"
+            />
+            Added
+          </span>
+        )}
```

Comment:
This could be simplified on toggling the text instead of the HTML tags too no?

```suggestion
       
          <span>
            <HeroIcon
              iconType="play"
              className="su-transition-colors su-inline"
            />
             {removeBtn ?  'Remove' : 'Added' }
          </span>
        
```